test/actions/filestorage.js

```diff
@@ -0,0 +1,55 @@
+/* global describe, before, after, it */
```

Comment:
this actually doesn't test the action, only the helper - better move to a separate file